composer.json

```diff
@@ -87,25 +87,45 @@
       "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"
     ],
     "test": [
-      "@php ./vendor/phpunit/phpunit/phpunit --no-coverage"
-    ],
-    "test10": [
-      "@php ./vendor/phpunit/phpunit/phpunit -c phpunit10.xml.dist --no-coverage"
+      "./scripts/run-phpunit.sh --no-coverage"
     ],
     "coverage": [
-      "@php ./vendor/phpunit/phpunit/phpunit"
+      "./scripts/run-phpunit.sh"
+    ],
+    "testserver:start": [
+      "./scripts/start-test-environment.sh"
+    ],
+    "testserver:stop": [
+      "./scripts/stop-test-environment.sh"
+    ],
+    "test:withserver": [
+      "@putenv REQUESTS_HTTP_PROXY=localhost:9002",
+      "@putenv REQUESTS_HTTP_PROXY_AUTH=localhost:9003",
+      "@putenv REQUESTS_HTTP_PROXY_AUTH_USER=test",
+      "@putenv REQUESTS_HTTP_PROXY_AUTH_PASS=pass",
```

Comment:
Could these be moved to a separate Composer script which is `@` referenced here ? (to avoid the duplication in the `coverage:withserver` script)